frontend/src/pages/profile/components/user-activity/styles.module.scss

```diff
@@ -0,0 +1,69 @@
+.title {
+  margin: 30px 0 15px;
+}
+
+.wrapper {
+  display: flex;
+  max-width: 830px;
+  overflow-x: auto;
+}
+
+.days {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding-right: 6px;
+}
+
+.months {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 6px;
+}
+
+.activity {
+  display: grid;
+  grid-template-rows: repeat(7, 25px);
+  grid-template-columns: repeat(auto-fit, 25px);
+  grid-auto-flow: column;
+  gap: 5px;
+}
+
+.activityItem {
+  background: rgba($color: #000000, $alpha: 10%);
+  aspect-ratio: 1 / 1;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.breakpoints {
+  display: flex;
+  align-items: center;
+  gap: 5px;
+  margin-top: 6px;
+}
+
+.breakpoint {
+  width: 11px;
+  height: 11px;
+}
+
+.breakpoint1 {
+  background: rgba($color: #000000, $alpha: 10%);
+}
+
+.breakpoint2 {
+  background: #9be9a8;
+}
+
+.breakpoint3 {
+  background: #40c463;
```

Comment:
Please create variables or use existing if any

Reply:
I'm not sure if i need to use variables here since these are temporary colors from github